src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for CustomerInsights under Azure Resource Manager.</Description>
-    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <VersionPrefix>0.1.1-preview</VersionPrefix>
```

Comment:
@tjlvtao why AssemblyVersion is different from <VersionPrefix>, that is what I wanted you address and make it consistent.
I am not sure how your code got into this state, you cannot have nuget package say one version and assemblyversion be different. And if you still want to do this way, please let me know what can be your reasons to have such different versions between Assembly and nuget version.